canonical/ubuntu.com
# Fix active nav item when multiple children start with active path

## Done

- Fix active nav item when multiple children url starts with the active path (e.g.: `advantage`, `advantage/users`)
  - add `aria-current="page"` to active navigation links for better accessibility

## Background
If there are multiple items in the navigation that start with the current URL we mark as active whichever comes first on the list and not evaluating other results.

This resulted in `/advantage` being marked as active when on page `/advantage/users`.

## Note
- this is the first time I wrote any `python`, if there's a better way of doing things, please do comment 🙏

## QA

- go to the demo site
- make sure navigation links are highlighted correctly - test pages with multiple children, e.g. /cube/study/packages/part-3


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/347

## Screenshots

## Before
![image](https://user-images.githubusercontent.com/7452681/138894731-4088c16c-bdb5-46f7-91a0-eb80177f77a0.png)

## After
![image](https://user-images.githubusercontent.com/7452681/138894678-fdeade2f-afbb-4337-b885-b7fecccf130b.png)
